tasks/run.js

```diff
@@ -196,6 +196,7 @@ module.exports = function (program, utils) {
 		.option('--key <file>', 'Specify a certificate key to use with HTTPS. Use with --https.')
 		.option('--local-apps <apps>', 'Specify extra apps that are running locally, as comma-seperated `[name]=[port]`, e.g. `service-worker=3001,front-page=3002`')
 		.option('-p --port <port>', 'Port to run the router through', 5050)
+		.option('--router <router>', 'Router to run using', 'next-router')
```

Comment:
I'm easy on this one, but think the first con listed is a good enough reason to just allow us to specify the full name of the router.